src/components/Footer/Footer.module.scss

```diff
@@ -0,0 +1,54 @@
+.footer_wrapper {
```

Comment:
incorrect folder name `Footer`.
folder name should start with small case. and while creating a folder, we should add an index file in the folder and export our component from there.
Please check this for ref `src/components/featureSlider/index.ts`